common/src/main/java/io/netty/util/HashedWheelTimer.java

```diff
@@ -253,6 +253,7 @@ public HashedWheelTimer(
         if (ticksPerWheel <= 0) {
             throw new IllegalArgumentException("ticksPerWheel must be greater than 0: " + ticksPerWheel);
         }
+        WORKER_STATE_UPDATER.set(this, WORKER_STATE_INIT);
```

Comment:
isn't this redundant because the variable int with no default value defined defaults to 0?

Reply:
Yes, but it is what the original code did.  (also, the JLS says that the default initialization of the field will always be zero, so I don't think it technically needs to be done).   I think it serves mostly as documentation.

Reply:
yeah I think its not needed but also not harms really as instances of `HashedWheelTimer` should not be created very frequently.